test/ecto/multi_test.exs

```diff
@@ -327,6 +327,19 @@ defmodule Ecto.MultiTest do
     ] = Ecto.Multi.to_list(multi)
   end
 
+  test "put" do
+    name = :halo
+    value = "statue"
+
+    multi =
+      Multi.new()
+      |> Multi.put(name, value)
+
+    assert multi.names == MapSet.new([name])
+    assert {:run, fun} = multi.operations[name]
```

Comment:
This is testing implementation details and it will break once we refactor. It is better to test that we get the “put” value on subsequent ops and as a result of transaction. :)